lib/swagger/generator.rb

```diff
@@ -18,6 +26,18 @@ module SwaggerGenerator
     # http://files.slatestudio.com/gG30
     ALLOW_TYPES  = %w(Object String Integer Array Date Mongoid::Boolean Symbol)
 
+    def swagger_options(options)
```

Comment:
This method doesn't look good, let's put default values back to the top of `generate_swagger_schemas ` and `generate_swagger_paths`.

So in the `swagger_options` you'll have:

`swagger_base_path = options[:base_path]`

And on the top of `generate_swagger_paths`:

`swagger_base_path ||= ''`